gammapy/spectrum/fit.py

```diff
@@ -226,9 +226,6 @@ def _predict_counts_helper(self, obs, model, forward_folded=True):
         else:
             raise ValueError('Predicted counts {}'.format(counts))
 
-        # Apply AREASCAL column
-        counts *= obs.on_vector.areascal
-
```

Comment:
Here it is assumed that the areascal will be always empty. This is probably fine. But one might issue a warning in case there is an areascal vector in the `PHACountSpectrum`, no?

Reply:
I put the application of ``areascal`` back. For us it will mostly be 1, but if it's there it seems better to use it than instead of raising a warning.